package.json

```diff
@@ -178,6 +178,7 @@
     "vue": "^3.2.20",
     "vue-loader": "^16.8.1",
     "vue-router": "^4.0.12",
+    "vue-template-compiler": "^2.6.10",
```

Comment:
Dont think this is relevant, we have moved to vue 3 where this shouldnt be required.

Reply:
I removed it, it was a leftover 😓